spec/models/hyrax/collection_type_spec.rb

```diff
@@ -1,4 +1,4 @@
-RSpec.describe Hyrax::CollectionType, type: :model do
+RSpec.describe Hyrax::CollectionType, clean_repo: true, type: :model do
```

Comment:
Yes, the clean_repo addresses the issue where if there are any collections of a type, then the type can't be saved.  Since the repo doesn't get cleaned between tests AND factory girl happily generates collection types with ids used previously, tests fail all over the place without the clean_repo.  Yuck.